client/src/app/login/login.component.html

```diff
@@ -31,5 +32,10 @@ <h1>{{ isManagementLogin ? "Management Login" : "Public Login" }}</h1>
       <button mat-button>Cancel</button>
       <button mat-button (click)="login()">Login</button>
     </div>
+    <div class="centered">
```

Comment:
Heyo, bit of a no no to put styling directly into HTML, makes it a nightmare when trying to debug styles later.
You can use mat error, just between the password & buttons like so.

>       "<mat-error class="centered">{{ errorText }}</mat-error>"

Reply:
Think I've got this right now?